linter_exclusions.yml

```diff
@@ -1341,6 +1341,28 @@ cosmosdb update:
     virtual_network_rules:
       rule_exclusions:
       - option_length_too_long
+cosmosdb restore:
+  parameters:
+    target_database_account_name:
+      rule_exclusions:
+        - option_length_too_long
+cosmosdb sql retrieve-latest-backup-time:
+  parameters:
+    target_database_account_name:
+      rule_exclusions:
+        - option_length_too_long
+cosmosdb database:
+  parameters:
+    db_resource_group_name:
+      rule_exclusions:
+        - option_length_too_long
+        - parameter_should_not_end_in_resource_group
+cosmosdb collection:
+  parameters:
+    db_resource_group_name:
+      rule_exclusions:
+        - option_length_too_long
+        - parameter_should_not_end_in_resource_group
```

Comment:
Why not add short optional names for those parameters?

Reply:
It'll be a breaking change then. cosmosdb database/collection apis already existing apis and if i shorten the name then it might need to be communicated to customer

Reply:
Same concern here. Can you keep the original name but add another shorter option names as alias? The shorter is just in CLI client side which will not break api definitions. Like `--db-resource-group-name` can have another option `--db-rg`

Reply:
Updated. Removed exclusions for option_length_too_long